test/suites/test-10-ecma.js

```diff
@@ -0,0 +1,32 @@
+define(['pumascript', 'esprima'], function(puma, esprima) {
+//section 10.1	
+  test("Use Strict global code", function () {
+    var result = puma.evalPuma("\'use strict\';var a = 1;
```

Comment:
Or you can properly concatenate the strings for better readability:
```
test("Use Strict global code", function () {
    var result = puma.evalPuma("'use strict';" +
        "var a = 1;" +
        "var b = 3;" +
        "console.log(a+b);"
    );
    ok(result.success && '4' === result.value, "Pass!");
});
```

Also note that there's no need to escape single quotes in line 4. You must escape it when you are using the same quotation marks for defining a string.